src/component/page/gjennomforing-page/GjennomforingListePage.tsx

```diff
@@ -28,37 +29,39 @@ export const GjennomforingListePage = (): React.ReactElement => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [])
 
-	const fetchGjennomforingerPromise = usePromise<AxiosResponse<Gjennomforing[]>>(
-		() => fetchTiltakGjennomforinger()
+	const fetchDeltakerOversiktPromise = usePromise<AxiosResponse<DeltakerOversikt>>(
+		() => fetchDeltakeroversikt()
 	)
 
-	if (isNotStartedOrPending(fetchGjennomforingerPromise)) {
+	if (isNotStartedOrPending(fetchDeltakerOversiktPromise)) {
 		return <SpinnerPage />
 	}
 
-	if (isRejected(fetchGjennomforingerPromise)) {
+	if (isRejected(fetchDeltakerOversiktPromise)) {
 		return <AlertPage variant="error" tekst="Noe gikk galt" />
 	}
+	
+	const deltakerOversikt = fetchDeltakerOversiktPromise.result.data
 
-	const alleGjennomforinger = fetchGjennomforingerPromise.result.data
+	if (deltakerOversikt.koordinator && deltakerOversikt.koordinator?.deltakerlister.length > 0) {
+		return (
+			<div className={styles.page} data-testid="gjennomforing-oversikt-page">
+				{ !env.isProd && deltakerOversikt.veileder && <MineDeltakerePanel veileder={deltakerOversikt.veileder}/> }
```

Comment:
Det kan være greit å ha en felles toggle for funksjonalitet (det finnes en `Toggle` klasse et sted) som gjør den samme `!env.isProd` sjekken. Det gjør det kanskje litt tryggere å skru den på, også kan man rydde opp når det trengs?

Reply:
Skal se om jeg finner det :)